Lib/fontTools/varLib/__init__.py

```diff
@@ -127,34 +124,16 @@ def _add_avar(font, axes):
 		# https://github.com/fonttools/fonttools/issues/1011
 		# TODO(anthrotype) revert this (and 19c4b37) when issue is fixed
 		curve = avar.segments[axis.tag] = {-1.0: -1.0, 0.0: 0.0, 1.0: 1.0}
+
 		if not axis.map:
 			continue
 
 		items = sorted(axis.map)
 		keys = [item[0] for item in items]
 		vals = [item[1] for item in items]
 
-		# Current avar requirements.  We don't have to enforce
-		# these on the designer and can deduce some ourselves,
-		# but for now just enforce them.
-		assert axis.minimum == min(keys)
-		assert axis.maximum == max(keys)
-		assert axis.default in keys
-		# No duplicates
-		assert len(set(keys)) == len(keys), (
-			f"{axis.tag} axis: All axis mapping input='...' "
-			"values must be unique, but we found duplicates."
-		)
-		assert len(set(vals)) == len(vals), (
-			f"{axis.tag} axis: All axis mapping output='...' "
-			"values must be unique, but we found duplicates."
-		)
-		# Ascending values
-		assert sorted(vals) == vals
```

Comment:
i know you are validating and raising exceptions beforehand, but keeping the assertions wouldn't do harm here. It's simply confirming what should always stay true